NetworkService/Tests/NetworkServiceTests/NetworkServiceTests.swift

```diff
@@ -0,0 +1,8 @@
+import XCTest
+@testable import NetworkService
+
+final class NetworkServiceTests: XCTestCase {
+    func testUseReturnTrue() {
```

Comment:
nit:

We use present simple tense, don't we?
```suggestion
    func testUseReturnsTrue() {
```

Reply:
yes, you right